backend/util/util.go

```diff
@@ -23,33 +23,41 @@ import (
 	"github.com/digitalbitbox/bitbox-wallet-app/util/errp"
 )
 
-// APIGet performs a HTTP Get call to a given endpoint and unmarshal the result
-// populating a given object.
-func APIGet(httpClient *http.Client, endpoint string, result interface{}) error {
+// APIGet performs a HTTP Get call to a given endpoint and unmarshal the result populating a given object.
+// Input params:
+// - `httpClient` which is used to perform the call
+// - `endpoint` the url for the endpoint to fetch
+// - `apikey` if not empty it is used as X-API-KEY header value
+// - `maxSize` indicates the max expected response size. If it exceeds the function returns an error
+// - `result` object that should be used to unmarshal the response body
+// Returns a pointer to the error code (if available) and possibly an error.
+func APIGet(httpClient *http.Client, endpoint string, apiKey string, maxSize int64, result interface{}) (*int, error) {
```

Comment:
I think it's more natural/idiomatic to return `int` not `*int` and just return `0` instead of `nil`.

Reply:
noted, thanks